x-pack/plugins/fleet/server/services/fleet_server/index.ts

```diff
@@ -113,18 +113,20 @@ export async function checkFleetServerVersionsForSecretsStorage(
   let hasMore = true;
   const policyIds = new Set<string>();
   let page = 1;
+  const perPage = 20;
   while (hasMore) {
     const res = await packagePolicyService.list(soClient, {
       page: page++,
-      perPage: 20,
+      perPage: 200,
```

Comment:
Why changing this and not using the constant defined on top?

Reply:
yeah that's a mistake